docs/users/cli-usage.md

```diff
@@ -5,12 +5,12 @@ Type `ddev` or `ddev -h`in a terminal windows to see the available ddev commands
 
 ## Quickstart Guides
 
-You can start using ddev by running just a few commands. Below are quickstart instructions WordPress, Drupal 7, Drupal 8, and TYPO3 CMS.
+These are quickstart instructions WordPress, Drupal 7, Drupal 8, TYPO3, and Backdrop.
```

Comment:
s/These/There/ - You want it to say "There are quickstart instructions for ..." right? So it needs the "for" also.  Why not make each of those a link?

Reply:
good catch! tnx, fixing.